lib/diagnostic.rb

```diff
@@ -8,14 +8,14 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = require 'lib/example.rb'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = const repl = require('repl')
 
```

Comment:
You would just enter `pry` to enter our favorite Ruby REPL!
A REPL is an acronym for Read–Eval–Print Loop or fancy words for a thing where we can interact with the language and code